Source/APIClient/Request.swift

```diff
@@ -34,15 +36,22 @@ public protocol Request {
 
 public extension Request {
 
+    var contentType: String {
+        switch self {
+        default:
+            return "application/json"
+        }
+    }
+
     var urlRequest: URLRequest {
         let baseURL = Foundation.URL(string: Self.baseURL)!
-        let url = Foundation.URL(string: path, relativeTo: baseURL)!
+        let url = Foundation.URL(string: path, relativeTo: baseURL) ?? baseURL
```

Comment:
Gotcha. In that case, does providing an empty string for the path result in an incorrect URL?